core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -1884,6 +1884,13 @@ public void testGrant()
                         QualifiedName.of("t"),
                         new PrincipalSpecification(PrincipalSpecification.Type.UNSPECIFIED, new Identifier("u")),
                         false));
+        assertStatement("GRANT UPDATE ON t TO u",
```

Comment:
You need to add also a test for REVOKE UPDATE

Reply:
Added REVOKE UPDATE to `TestSqlParser#testRevoke`